editing/fx/passes/eps.py

```diff
@@ -406,6 +417,10 @@ def run_pass(self, gm : fx.GraphModule):
                             print("[AnnotateEpsPass] Mismatching input n_levels in node with no n_levels_out propagation function! n_levels propagation will likely be wrong!")
                             print(f"                    -> Node: {node.name}, Key: {k}, n_levels_in: {node_in_levels}")
                             if (self.verbose): print(f"[AnnotateEpsPass] Using identity n_level propagation on node with op {node.op}, target {node.target}!")
+
+                        if node.op == "call_function" and node.target == torch.cat:
+                            node_in_levels = [inp.meta['quant'].n_levels_in for inp in node.args[0]]
```

Comment:
Please extract this into a more proper format - maybe a function that returns node_in_levels at the top of this code block rather than hacking this in here.

Reply:
Same here, why `signed_in` and not `signed_out`?

Reply:
@Scheremo: Done [here](https://github.com/pulp-platform/quantlib/pull/15/commits/09d2983fd33738ad3b193fa3917588587c075cad)

@Xeratec: Done [here](https://github.com/pulp-platform/quantlib/pull/15/commits/ed0d20c296124c28c4be0d9e9f3b2e0c5a5f63cf)